packages/react-server/docs/why-react-server.md

```diff
@@ -151,6 +152,22 @@ a few milliseconds. Browsers are well equipped to render partial pages, and
 the user will be shown some content while the primary part of the page is
 still loading.
 
+## Streaming client initialization
+
+Beyond simple streaming of the HTML content, `react-server` also initializes
+the `React` components in the browser _as they arrive_.  You tell
```

Comment:
Yeah!  That was the flip side to dropping the synchronous render (which turned out to be pretty confusing for devs and the source of tricky bugs).

Now data is either available for the server render or it's not.  We've got a helper in our store class that makes it easy to configure which values will block the server render and which will get populated _after_ the initial client render (to avoid checksum mismatch).  There's nothing in between.

Haven't figured out how to split that scheduling/filtration out of our stores yet to make it open.  Our stores are based on reflux, and I wouldn't want to impose that as a requirement for core functionality in `react-server`.